src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPoolPreAllocatedOverlapped.WindowsThreadPool.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+namespace System.Threading
+{
+    public sealed partial class PreAllocatedOverlapped : IDisposable, IDeferredDisposable
```

Comment:
I suggest renaming this file to `PreAllocatedOverlapped.WindowsThreadPool.cs` for consistency